app/models/order.rb

```diff
@@ -0,0 +1,8 @@
+class Order < ApplicationRecord
+  has_many :order_items
+
+  def order_total
+
+  order_items.joins(:product).sum('products.price')
+  end
+end
```

Comment:
Please be mindful of the importance of code formatting

Reply:
Yes, we will.